src/ui/public/agg_types/buckets/histogram.js

```diff
@@ -95,6 +96,12 @@ export const histogramBucketAgg = new BucketAggType({
               min: _.get(resp, 'aggregations.minAgg.value'),
               max: _.get(resp, 'aggregations.maxAgg.value')
             });
+          })
+          .catch(() => {
+            toastNotifications.addWarning(`
+              Unable to retrieve max and min values to auto-scale histogram buckets. 
+              This may lead to poor visualization performance.
```

Comment:
Is it a performance issue? Unless I'm misunderstanding, it seems like it's more of a problem with the veracity of the visualization's results. Would "This may lead to incorrect visualization results" be more accurate?

Reply:
it's actually to increase the size of the buckets in case the given minimum interval causes too many buckets, see [this comment](https://github.com/elastic/kibana/issues/17662#issuecomment-380549354). when I tested using a rollup field that doesn't have min/max after adding the `catch`, I get a visualization that looks like this: 

![image](https://user-images.githubusercontent.com/1965714/46363111-f8cc5e00-c626-11e8-91b5-dbb2398b43d8.png)

if it had min/max, the interval would have been increased to decrease the number of bars. this example loaded fine for me, but I imagine a greater number of bars could be problematic on certain machines. hence performance issue 🙂